### ISSUE TITLE
UI collapses when writing in TextFormField

### ISSUE THREAD

**Issue by virenvvarasadiya** [flutter/flutter#30043]
I am trying to make simple UI where there is a column with three elements. The first one is a listview, the second one is a TextFormField and last one is a container. When I try to type in the TextFormField then the bottom container also replaces which I don't understand.

Here is my code to help you understand my problem more clearly.

	class demo extends StatefulWidget {
	  @override
	  _demoState createState() => _demoState();
	}

	class _demoState extends State<demo> {
	  @override
	  Widget build(BuildContext context) {
		return Scaffold(
		  appBar: AppBar(title: Text("demo"),),
		  body: new Column(
			children: <Widget>[
			  Expanded(
				child: new ListView.builder(
				  itemCount:1000,
				  itemBuilder: (context,index){
					return Container(
					  padding: EdgeInsets.all(10.0),
					  height: 25.0,
					  color: Colors.grey,
					  child: Text(index.toString()),
					);
				  }
				),
			  ),
			  new TextFormField(
				decoration: InputDecoration(
				  border: OutlineInputBorder(),
				  hintText: "Enter Demo",
				  hintStyle: TextStyle(color: Colors.grey),
				),
			  ),
			  new Container(
				height: 20.0,
				child: Text("Demo Text"),
			  )
			],
		  ),
		);
	  }
	}

**Comment by Kartik1607** [comment]
Could you please specify problem more? When I tapped on the textfield the keyboard popped up. Do you not want the container to be shown when keyboard is open, and only form field to be visible above keyboard?

**Comment by virenvvarasadiya** [comment]
yes exactly, when i type in textfield i do not want to show container. as it show's right now.
